.azure-pipelines/windows-visual-studio.yml

```diff
@@ -0,0 +1,58 @@
+steps:
+  - script: |
+      :: Use Windows CRLF line endings for checked-out text files
+      git config --global core.autocrlf true
+      set
+    displayName: Print environment variables
+  - checkout: self
+    fetchDepth: 1
+  - script: |
+      bash --version
+      sh --login .azure-pipelines/windows-visual-studio.sh ldc
+    displayName: Download LDC
+  - script: |
+      sh --login .azure-pipelines/windows-visual-studio.sh visuald
+    displayName: Download VisualD
+  - script: |
+      sh --login .azure-pipelines/windows-visual-studio.sh dm_make
+    displayName: Download DigitalMars make
+  - script: |
+      sh --login .azure-pipelines/windows-visual-studio.sh clone
+    displayName: Clone druntime + phobos
+  - script: |
+      echo on
+      call "%VSINSTALLDIR%\VC\Auxiliary\Build\vcvarsall.bat" %ARCH%
+      set LDC_DIR=ldc2-%LDC_VERSION%-windows-multilib
+      set VISUALD_INSTALLER=VisualD-%VISUALD_VER%.exe
+      set PWD=%cd%
+      FOR /F "tokens=* USEBACKQ" %%F IN (`where cl.exe`) DO (SET CC=%%~fsF)
+      FOR /F "tokens=* USEBACKQ" %%F IN (`where make.exe`) DO (SET GNU_MAKE=%%~fsF)
+
+      .\%VISUALD_INSTALLER% /S
+      REM configure DMD path
+      reg add "HKLM\SOFTWARE\DMD" /v InstallationFolder /t REG_SZ /d "%PWD%\dmd2" /reg:32 /f
+      REM configure LDC path
+      reg add "HKLM\SOFTWARE\LDC" /v InstallationFolder /t REG_SZ /d "%PWD%\%LDC_DIR%" /reg:32 /f
+
+      REM disable link dependencies monitoring, fails on AppVeyor server
+      #reg add "HKLM\SOFTWARE\Microsoft\VisualStudio\15.0\ToolsOptionsPages\Projects\Visual D Settings" /v optlinkDeps /t REG_DWORD /d 0 /reg:32 /f
+
+      REM build via VS projects with LDC
+      cd src
+      REM unfortunately, multilib version of LDC and Visual D don't work together seamlessly (anymore?)
+      set LDC_ARGS="/p:ExternalLinkerOptions=phobos2-ldc.lib druntime-ldc.lib"
+      set LDC_ARGS=%LDC_ARGS% /p:DCompiler=LDC
+      msbuild /target:dmd /p:Configuration=Release /p:Platform=Win32 %LDC_ARGS% vcbuild\dmd.sln
+
+      REM Check: build druntime unittests and dmd test suite
+      cd "%PWD%\..\druntime"
+      set DM_MAKE=%PWD%\dm\bin\make.exe
+      set DMD=%PWD%\dmd\generated\Windows\Release\Win32\dmd.exe
+      "%DM_MAKE%" -f win64.mak MODEL=32mscoff DMD="%DMD%" VCDIR="%VCINSTALLDIR%." "CC=\"%CC%\"" SDKDIR=unused target unittest32mscoff
```

Comment:
This seems to invoke mingw32-make in sub processes. You will need to add `"MAKE=%DM_MAKE%"` to the command line.

You should also be able to remove the escaped quotes `\"` from the CC variable now.